app/workers/ons_download_worker.rb

```diff
@@ -0,0 +1,42 @@
+require "open-uri"
+require "zip"
+
+class OnsDownloadWorker < OnsBaseWorker
+  # Example URL: https://www.arcgis.com/sharing/rest/content/items/a2f8c9c5778a452bbf640d98c166657c/data
+  # retrieved by visiting https://geoportal.statistics.gov.uk/search?collection=Dataset&sort=-created&tags=all(PRD_ONSPD)
+  # clicking on the first search result and then copying the link from the download button.
+
+  DATAFILE_REGEX = /\AData\/multi_csv\/ONSPD_(.*)_UK_(.*).csv\z/
```

Comment:
Is the filename structure likely to change? An alternative may be just to open the multi_csv dir and run through all the csv's (.csv) within

Reply:
Nice, thanks Keith!